content/docs/user-guide/assets/asset-processor/settings.md

```diff
@@ -0,0 +1,46 @@
+| additionalScanFolders           | Used with `dependencyScanPattern` to farther filter the scan.                                                                                                                                               |                                                                                                 |
+| debugOutput                     | When enabled, builders that support it will output debug information as product assets. Used primarily with scene files.                                                                                    |                                                                                                 |
+| dependencyScanMaxIteration      | Used to limit the number of recursive searches per line when running dependencyScanPattern.                                                                                                                 | Default is 800.                                                                                 |
+| dependencyScanPattern (dsp)     | Scans assets that match the given pattern for missing product dependencies.                                                                                                                                 |                                                                                                 |
+| enableQueryLogging              | Enables logging database queries.                                                                                                                                                                           |                                                                                                 |
+| fileDependencyScanPattern (fsp) | Used with `dependencyScanPattern` to farther filter the scan.                                                                                                                                               |                                                                                                 |
+| help (h)                        | Displays all available commands and their description.                                                                                                                                                      |                                                                                                 |
+| sortJobsByDBSourceName          | When enabled, sorts pending jobs with equal priority and dependencies by database source name instead of job ID. Useful for automated tests to process assets in the same order each time.                  |                                                                                                 |
```

Comment:
this is the default behavior now, and this flag has been removed

Reply:
@AMZN-stankowi Note this is targetting the 2205 release / main and not development.